.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -27,9 +27,7 @@ locals {
   # TODO: Assess whether this is safe for Portal API as well. Trying 1 worker in rdev portal backend containers, to minimize use of memory by TileDB (allocates multi-GB per process)
   # Note: keep-alive timeout should always be greater than the idle timeout of the load balancer (60 seconds)
   backend_workers              = var.backend_workers 
-  backend_cmd                  = ["gunicorn", "--worker-class", "gevent", "--workers", "${local.backend_workers}",
-    "--bind", "0.0.0.0:5000", "backend.api_server.app:app", "--max-requests", "10000", "--timeout", "180",
-    "--keep-alive", "61", "--log-level", "info"]
+  backend_cmd                  = []
```

Comment:
Since we are using *container_init.sh*, does that mean all ECS tasks will only have a single worker?

Reply:
oh! Nice catch again! I will revert this back. I was copying from the [POC](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3610/files#diff-537dc26b6e537cf0bd0dc71ff411df19fd373540ddd6c0f90160f777f17a7591)